src/components/Loader/Loader.module.scss

```
@@ -23,7 +23,7 @@
   animation: rotation 4s linear 0s infinite; 
   left: 50%;
   top: 50%;
-  margin-top: 50px;
+  margin: 50px 0 50px 0;
```

аналогично и короче:
```suggestion
  margin: 50px 0;
```